Makefile.am

```diff
@@ -56,7 +31,40 @@ service_DATA = \
 	search-provider/com.endlessm.EknServices.SearchProviderV1.service \
 	$(NULL)
 
+search-provider/eks-search-provider-dbus.h search-provider/eks-search-provider-dbus.c: search-provider/eks-search-provider-dbus.xml Makefile.am
+	$(AM_V_GEN) $(GDBUS_CODEGEN) \
+	--interface-prefix=org.gnome.Shell \
+	--c-namespace Eks \
+	--generate-c-code search-provider/eks-search-provider-dbus \
+	$<
+
+search-provider/eks-knowledge-app-dbus.h search-provider/eks-knowledge-app-dbus.c: search-provider/eks-knowledge-app-dbus.xml Makefile.am
+	$(AM_V_GEN) $(GDBUS_CODEGEN) \
+	--interface-prefix=com.endlessm. \
+	--c-namespace Eks \
+	--generate-c-code search-provider/eks-knowledge-app-dbus \
+	$<
+
```

Comment:
Ah, now I know what caused this: you need `$(MKDIR_P) $(@D) &&` in these two rules before gdbus-codegen.